guides/novu-fcm-web.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: 'How to use FCM with Novu'
+description: 'Send FCM push notifications in a React app using Novu'
+icon: 'circle-6'
+---
+
+## Introduction
+In this guide, we'll learn how to send FCM push notifications in a web app using Novu. But before exploring the actual code, let’s understand what a push notification is and how it works.
+You can find the entire code (frontend as well as backend) for this app [here](https://github.com/novuhq/).
+<br/>
+Push notifications are notifications that are 'pushed' to a user's devices whether the user is using the app or not. We'll be using Firebase Cloud Messaging or FCM integration of Novu to send these notifications. This guide will be broken down into three parts:
+1. Setting up Novu
+2. Setting up Firebase
+3. Adding Firebase to the Frontend, and
+4. Adding Novu to the Backend
+
+## Setting up Firebase
+1. Create a Firebase account if you don't already have one. Then, create a new project and all the relevant details.
+<Frame caption="Create a new Firebase project"><img src="/images/fcm-web-push/project.png" /> </Frame>
+2. Once your project has been created, you'll be greeted with this welcome screen and asked to add Firebase to your app, so let's do that.
+<Frame caption="Now, we need to add firebase to our app"><img src="/images/fcm-web-push/add.png" /> </Frame>
+3. Because we're creating a web app here, we'll choose the 'web' option.
```

Comment:
We shouldn't start a sentence with 'Because'. So you can rewrite this to be:

We'll choose the `web` option because we're creating a web app.